corehq/apps/accounting/payment_handlers.py

```diff
@@ -335,28 +353,40 @@ def create_charge(self, amount, card=None, customer=None):
         )
 
     def update_credits(self, payment_record):
-        self.credit_line = CreditLine.add_credit(
-            payment_record.amount, account=self.account, subscription=self.subscription,
-            product_type=self.product_type, feature_type=self.feature_type,
-            payment_record=payment_record,
-        )
+        for feature in self.features:
+            feature_amount = feature['amount']
+            if feature_amount >= 0.5:
```

Comment:
what happens if ```feature_amount < 0.5```?  Consider logging an error to sentry instead of failing silently